code/data_structures/hash_table/hashtable.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
The hashmap implementation should adhere to OOP design principles and should have basic functions such as `put()`, `get()`, `size()` and `remove()` 👍

Reply:
I ll do it bro thnx